arp1/index.mjs

```diff
@@ -139,6 +146,14 @@ function bang() {
   }
 }
 
+function resetArp() {
+  heldNotes = []; // what the user is pressing down now
+  heldVelos = []; // save the velocities and loop through them
+  sequence = []; // this is what will play, in order
+  sIndex = 0; // the currently playing note in the sequence
+  previousNote = null; // value of the note
+}
```

Comment:
Nice. This is helpful documentation too.

Reply:
This could be a nice pattern actually. Since these are the same as the variable initializations at the top, we could even add some global initial-value constants and have both the variable initializations and resetArp reference the same constants?